app/models/email_message.rb

```diff
@@ -6,8 +6,8 @@ class EmailMessage < Ahoy::Message
   # reasons to define behavior here, similar to how we use the Tag model.
   def body_html_content
     doctype_index = content.index("<!DOCTYPE")
-    closing_html_index = content.index("</html>") + 6
-    content[doctype_index..closing_html_index]
+    closing_body_index = content.index("</body>") + 6
+    content[doctype_index..closing_body_index]
```

Comment:
This code had to be changed because Ahoy is no longer doing HTML processing with Nokogiri: it's treating the email as a fragment: https://github.com/ankane/ahoy_email/compare/v1.1.1...v2.0.2#diff-68d3d25e705f575b9bd84cc277469ed00524fa4e38e69d5291310887eea06bcbR57